projects/step-frontend/src/lib/modules/timeseries/chart/tooltip-plugin.ts

```diff
@@ -116,48 +267,42 @@ export class TooltipPlugin {
             }
           }
           if (yPoints.length === 0) {
-            overlay.style.zIndex = '-1';
+            tooltip.style.zIndex = '-1';
             return; // there is no data to show
           } else {
-            overlay.style.zIndex = '1000';
+            tooltip.style.zIndex = '1000';
           }
-          overlay.innerHTML = '';
+          tooltip.innerHTML = '';
           yPoints.forEach((point) => {
-            let rowElement = this.createRowElement(point, settings.yAxisUnit);
-            overlay.appendChild(rowElement);
+            let rowElement = createRowElement(point, settings.yAxisUnit);
+            tooltip.appendChild(rowElement);
           });
           if (yPoints.length < allSeriesLength) {
             if (elipsisBefore) {
-              let dots = document.createElement('div');
-              dots.classList.add('dots');
-              dots.textContent = '...';
-              overlay.prepend(dots);
+              tooltip.prepend(createDotsSeparator());
             }
             if (elipsisAfter) {
-              let dots = document.createElement('div');
-              dots.classList.add('dots');
-              dots.textContent = '...';
-              overlay.appendChild(dots);
+              tooltip.appendChild(createDotsSeparator());
             }
           }
           if (summaryRow) {
-            let summaryElement = this.createRowElement(summaryRow);
+            let summaryElement = createRowElement(summaryRow);
```

Comment:
In case if value defined only once and never changed further - use `const` instead of `let`.
This remark relates to other places too